Makefile.common

```diff
@@ -14,6 +14,9 @@
 # Executables
 ###########################################################################
 
+COQBIN:=$(PWD)/bin/
+export COQBIN
+
```

Comment:
This looks very suspicious, in particular for non-local builds.

Reply:
Is this still needed?

Reply:
It's so we can override it in gitlab: `make COQBIN=...`

Reply:
But my worry here is that the path may not be correct, right? In non-local builds we should use `DESTDIR`?

Reply:
I'm against this change. If you need `$COQBIN` to hold a good value, export it in the CI script.

Reply:
I agree with Enrico, this looks fishy somehow.